pkg/cloudprovider/aws/apis/v1alpha1/provider.go

```diff
@@ -34,6 +34,9 @@ type AWS struct {
 	// TypeMeta includes version and kind of the extensions, inferred if not provided.
 	// +optional
 	metav1.TypeMeta `json:",inline"`
+	// AMIFamily is the AMI family that instances use.
+	// +optional
+	AMIFamily string `json:"amiFamily"`
```

Comment:
Since this is optional, let's make it a `*string` and `json:"amiFamily,omitempty"`